backend/apps/profile_info/factories.py

```diff
@@ -0,0 +1,14 @@
+import factory
+from apps.users.models import BaseUser
+
+class BaseUserFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = BaseUser
+
+    username = factory.Faker("user_name")
```

Comment:
There is already a factory for BaseUser - users/factories.py. This can be deleted.